lib/git-prompt-server.js

```diff
@@ -25,25 +26,41 @@ export default class GitPromptServer {
     // TODO: [mkt] Windows?? yes.
     this.promptForInput = promptForInput;
     const windows = process.platform === 'win32';
-    this.tmpFolderPath = await getTempDir('github-');
+    this.tmpFolderPath = await getTempDir({
+      dir: windows ? os.tmpdir() : '/tmp',
```

Comment:
Why is this hard-coded outside of Windows instead of just using os.tmpdir?  Seems it would ignore the environment https://github.com/nodejs/node-v0.x-archive/blob/master/lib/os.js#L47